### ISSUE TITLE
Java Plugin Example not working

### ISSUE THREAD

**Issue by pwonson** [rundeck/docs#1006]
**Describe the bug**
https://github.com/clofresh is used in the developing a java plugin example section and the build no longer works.

If we want to use this example we should pull it into our own support repository.

**Source page**
[Please include the link or name for the page of the documentation which has the issue.
](https://docs.rundeck.com/docs/learning/howto/java-plugin.html#developing-a-custom-rundeck-java-plugin)

**Comment by fdevans** [comment]
We are working on a comprehensive set of new examples and starter dev content.  Thanks for calling this out.  We'll update once everything is ready.

**Comment by fdevans** [comment]
Thanks for the report. As part of a cleanup of our documentation issue tracker, we're closing issues older than 12 months — the docs site has been substantially reorganized and rewritten since older reports were filed, so we can't be confident this still applies to the current content. This isn't a judgment that the report was invalid.

If this is still an issue on the current docs site (docs.rundeck.com), please open a new issue linking the specific page and describing what's wrong. Please don't reopen this issue — a fresh issue keeps things properly tracked against the current site.

Thanks for your patience,
The Rundeck Team
